src/h5web/providers/jupyter/api.ts

```diff
@@ -108,12 +109,14 @@ export class JupyterApi implements ProviderAPI {
     }
 
     if (isDatasetResponse(response)) {
-      const { type, shape: dims } = response;
+      // `type` is the type of the fetched entity
```

Comment:
I feel this comment adds confusion 😅 -- the renaming in the destructuring and the second comment are sufficient.